Riot/Modules/Room/RoomViewController.m

```diff
@@ -3331,9 +3331,15 @@ - (void)showAdditionalActionsMenuForEvent:(MXEvent*)selectedEvent inCell:(id<MXK
                 
                 [self cancelEventSelection];
                 
-                NSArray *activityItems = @[selectedComponent.textMessage];
+                UIActivityViewController *activityViewController = nil;
+                if (selectedEvent.location) {
+                    activityViewController = [self.delegate roomViewController:self locationShareActivityViewControllerForEvent:selectedEvent];
```

Comment:
Would it be in scope for this PR to add a `shareEvent` method to `RoomCoordinator`? It feels weird having the view controller ask its coordinator to get another view controller from a different coordinator for it to present itself.

Reply:
I thought about it but don't want to expose the cell for setting the `sourceView` and `sourceRect` on the popover. I feels like something that should go in a bigger refactor.

Reply:
Makes sense to me 👍